src/config/entities/postgres.config.ts

```diff
@@ -0,0 +1,49 @@
+import { readFileSync } from 'fs';
+import type { PostgresConnectionOptions } from 'typeorm/driver/postgres/PostgresConnectionOptions';
+
+interface IPostgresEnvConfig {
+  type: 'postgres';
+  host: string;
+  port: string;
+  username: string;
+  password: string;
+  database: string;
+  ssl: {
+    enabled: boolean;
+    caPath: string | undefined;
+    requestCert: boolean;
+    rejectUnauthorized: boolean;
+  };
+}
+
+export const postgresConfig = (
+  postgresEnvConfig: IPostgresEnvConfig,
+): PostgresConnectionOptions => {
+  const isCIContext = process.env.CI?.toLowerCase() === 'true';
+  const isSslEnabled = !isCIContext && postgresEnvConfig.ssl.enabled;
+  const postgresCa =
+    isSslEnabled &&
+    postgresEnvConfig.ssl.caPath &&
+    postgresEnvConfig.ssl.caPath.length > 0
+      ? readFileSync(postgresEnvConfig.ssl.caPath, 'utf8')
+      : undefined;
+
+  return {
+    type: 'postgres',
+    synchronize: false,
+    host: postgresEnvConfig.host,
+    port: ~~postgresEnvConfig.port,
+    username: postgresEnvConfig.username,
+    password: postgresEnvConfig.password,
+    database: postgresEnvConfig.database,
+    migrations: ['dist/migrations/*.js'],
+    // logging: !isCIContext,
```

Comment:
If we can't have CI logging, I'd remove this.

Reply:
I would say let's keep it, it's like a TODO. I should have been more clear by adding a TODO.